extension/tests/component/__snapshots__/createOrder.test.ts.snap

```diff
@@ -26,7 +26,7 @@ exports[`Create Order Happy Path Should return 201 when mollie order created fro
 exports[`Create Order Happy Path Should return 201 when mollie order created from cart with both Line Items and Custom Line Items 3`] = `
 {
   "action": "setKey",
-  "key": "ord_ca1j7q",
+  "key": "ord_1_ca1j7q",
```

Comment:
I assume the `_` notation is because the target API doesn't support the `.` charater in the `key`field?

Reply:
Exactly, so our connector read ord_1.ca1j7q and convert it to ord_1_ca1j7q before storing into the OrderKey